### PR TITLE
ECC-1133 Change UK VAT Page body text & title

### PR DESCRIPTION
**Pull request by vigneshwaripandi**

### CODE REVIEW

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/views/vat_details.scala.html [hunk 1]
```diff
@@ -24,11 +24,15 @@
...
+@import uk.gov.hmrc.eoricommoncomponent.frontend.config.AppConfig
 
-@this(inputDate_di: inputDate, layout_di: layout, govukButton: GovukButton, inputText: helpers.inputText, errorSummary: helpers.errorSummary)
+@this(inputDate_di: inputDate, layout_di: layout, govukButton: GovukButton, inputText: helpers.inputText, errorSummary: helpers.errorSummary, appConfig:AppConfig)
 
 @(vatForm: Form[VatDetails], isInReviewMode: Boolean, service: Service)(implicit request: Request[_], messages: Messages)
 
+@vatUKLink = {
+<a class="govuk-link" href="@{appConfig.vatUKDetails}" target="_blank" rel="noopener noreferrer">@messages("cds.subscription.vat-details.intro-text.link")</a>
+}
```
Comment (NandanaRaghu) [inline app/uk/gov/hmrc/eoricommoncomponent/frontend/views/vat_details.scala.html:36]: Can u add the test for view for the new content

Comment (vigneshwaripandi) [reply app/uk/gov/hmrc/eoricommoncomponent/frontend/views/vat_details.scala.html:36]: I have added the tests for this view. Please review.

File: conf/messages.en [hunk 2]
```diff
@@ -855,14 +855,15 @@ cds.subscription-details.amend.legal-entity.user.header=Edit legal entity
...
+cds.subscription.vat-details.heading=Your UK VAT details
+cds.subscription.vat-details.intro-text=You can find your VAT details on your VAT registration certificate. To view this login to your tax account and select ''More VAT details''. Select the ''View VAT certificate'' link under the heading ''Help with tax''.
+cds.subscription.vat-details.intro-text.link=Find your VAT details (opens in new tab).
+cds.subscription.vat-details.postcode=VAT registration address postcode
 cds.subscription.vat-details.postcode.required.error=Enter a valid postcode of your VAT registration address
 cds.subscription.vat-details.postcode.length.error=The postcode must be a maximum of 8 characters
 cds.subscription.vat-details.vat-number=VAT number
 cds.subscription.vat-details.vat-effective-date=Effective VAT date
-cds.subscription.vat-details.vat-effective-date.hint=For example, 31 3 1980
+cds.subscription.vat-details.vat-effective-date.hint=For example, 01 11 2007
```
Comment (NandanaRaghu) [inline conf/messages.en:866]: do we need to change the vat effective date for cy as well

Comment (vigneshwaripandi) [reply conf/messages.en:866]: I have added the date to welsh messages as well now.

**Review by NandanaRaghu — APPROVED** [review]
LGTM

**Pull request merged**